### PR TITLE
feat(KtButton): Add iconPosition prop defaulted to left

### PR DESCRIPTION
**Pull request by Juli0GT**
Add iconPosition prop, to allow users to add icon to the right of the label.

Default value is left.

### CODE REVIEW

File: packages/kotti-ui/source/kotti-button/KtButton.vue [hunk 1]
```diff
@@ -8,11 +8,12 @@
 			@click="handleClick"
 		>
 			<i v-if="isLoading" class="kt-circle-loading" />
-			<i v-else-if="icon !== null" class="yoco" v-text="icon" />
+			<i v-else-if="hasIconLeft" class="yoco" v-text="icon" />
 			<span v-if="hasSlot">
 				<slot />
 			</span>
 			<span v-else-if="label !== null" v-text="label" />
+			<i v-if="hasIconRight && !isLoading" class="yoco" v-text="icon" />
```
Comment (carsoli) [inline packages/kotti-ui/source/kotti-button/KtButton.vue:16]: technically for the left Icon we remove the icon and replace it with the loading icon (loading comes on left)

but you don't need the `!isLoading` for the right Icon

File: packages/documentation/pages/usage/components/button.vue [hunk 2]
```diff
@@ -65,18 +65,21 @@
...
     	<KtButton type="primary" class="mr-4">Edit button</KtButton>
     	<KtButton type="primary" icon="edit" label="Edit Button" class="mr-4" />
+    	<KtButton type="primary" icon="edit" label="Right Icon Button" iconPosition="right"/>
     	<KtButton type="primary" icon="edit" helpText="This is an icon button"/>
     </div>
 
     * **Label only:** Used in most cases.
     * **Icon and label:** Use when you need to catch the user's attention.
     * **Icon only:** Use when you have limited space, such as when the page needs to fit on a mobile device, and a single icon is enough to convey the meaning,
+    * **iconPosition** prop can be used to place icon to the right of the label. Is left by defaul.
```
Comment (santiagoballadares) [inline packages/documentation/pages/usage/components/button.vue:75]:
```suggestion
    * **iconPosition** prop can be used to place icon to the right of the label. Is left by default.
```

File: packages/kotti-ui/source/kotti-button/KtButton.vue (hunk 1 above)
Comment (santiagoballadares) [reply packages/kotti-ui/source/kotti-button/KtButton.vue:16]: @carsoli is it agreed that loading is rendered always on the left side of the button regardless of the icon position?
or should it be placed in the same position as the icon?

Comment (Juli0GT) [reply packages/kotti-ui/source/kotti-button/KtButton.vue:16]: Good point @santiagoballadares probably this is a question for Alberto.

Comment (Juli0GT) [reply packages/kotti-ui/source/kotti-button/KtButton.vue:16]: @santiagoballadares just asked Alberto he agreed to keep it on the left.

**Review by santiagoballadares — APPROVED** [review]

**Pull request merged**